### PR TITLE
Adding jobsub_cleanup_cred script

### PR DESCRIPTION
**Pull request by marcmengel**
Tool for users who have hosey credentials stuck on the credds to clean them all out so they can resubmit.

### CODE REVIEW

File: bin/jobsub_cleanup_cred [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+sys.path.append(os.path.join(PREFIX, "lib"))
+
+import fake_ifdh
+from creds import get_creds, CredentialSet
+import get_parser
+from condor import get_schedd_names
+
+notes = """
+
+BEARER_TOKEN_FILE=/tmp/bt_token... _condor_COLLECTOR_HOST=dunegpcoll02.fnal.gov _condor_CREDD_HOST=dunegpschedd02.fnal.gov condor_store_cred qery-oauth
```
Comment (shreyb) [inline bin/jobsub_cleanup_cred:45]: Should this be `query-oauth` at the end?

File: bin/jobsub_cleanup_cred [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+A credential was stored and is valid.
+
+Credential info:
+dune.top = 1681431891
+dune.use = 1682976510
+dune_ab0f4c39c0.top = 1682090936
+dune_ab0f4c39c0.use = 1682976330
+and:
+BEARER_TOKEN_FILE=/tmp/bt_token... _condor_COLLECTOR_HOST=dunegpcoll02.fnal.gov _condor_CREDD_HOST=dunegpschedd02.fnal.gov condor_store_cred delete-oauth -s credname
```
Comment (shreyb) [inline bin/jobsub_cleanup_cred:59]: Please remove the hostnames here.  FNAL security policy, apparently.

File: bin/jobsub_cleanup_cred [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        tname, handle = tname.split("_")
+        cmd = f"condor_store_cred delete-oauth -s {tname} -H {handle}"
+    else:
+        cmd = f"condor_store_cred delete-oauth -s {tname}"
+    print(f"Running: '{cmd}' against {schedd}:")
+    os.system(cmd)
+
+
+def main():
+    parser = get_parser.get_jobid_parser(add_condor_epilog=True)
```
Comment (shreyb) [inline bin/jobsub_cleanup_cred:97]: Scope increase request:

Could we have a --test or --dryrun argument that would tell the user which credentials on which schedd would get deleted, but not actually do the delete?

**Review by shreyb — CHANGES_REQUESTED** [review]
I generally like this.  Just a couple of comments and one request.

File: bin/jobsub_cleanup_cred (hunk 3 above)
Comment (marcmengel) [reply bin/jobsub_cleanup_cred:97]: We can use -n for it and not add another flag... yes.

**Review by shreyb — APPROVED** [review]
LGTM

**Pull request merged**